cli/src/main.rs

```diff
@@ -86,8 +87,34 @@ fn execute<S, I>(command: I) -> Result<String, Error> where I: IntoIterator<Item
 	};
 
 	info!(target: "bridge", "Starting listening to events");
-	let bridge = create_bridge(app_ref, &database).and_then(|_| future::ok(true)).collect();
-	event_loop.run(bridge)?;
+	let bridge = create_bridge(app_ref.clone(), &database).and_then(|_| future::ok(true)).collect();
+	let mut result = event_loop.run(bridge);
+	loop {
+		result = match &result {
+			&Err(Error(ErrorKind::Web3(web3::error::Error(web3::error::ErrorKind::Io(ref e), _)), _)) if e.kind() == ::std::io::ErrorKind::BrokenPipe => {
+			    warn!("Connection to Parity has been severed, attempting to reconnect");
```

Comment:
Bridge can work with geth too. So does it make sense to use more common statement like 'Connection to the node ...'?

Reply:
Sure. I was just printing this because the repository is called `parity-bridge` :)

Reply:
Understood. But it is not from Parity anymore :) our repo's name is more legacy matter than real restriction. Don't you mind to change the message?